c++/CMakeLists.txt

```diff
@@ -82,11 +82,40 @@ else()
 endif()
 
 # Source =======================================================================
-
+include(CapnProtoMacros)
 add_subdirectory(src)
 
 # Install ======================================================================
 
+include(CMakePackageConfigHelpers)
+write_basic_package_version_file(
+  "${CMAKE_CURRENT_BINARY_DIR}/CapnProtoConfigVersion.cmake"
+  VERSION ${VERSION}
+  COMPATIBILITY AnyNewerVersion
+)
+set(ConfigPackageLocation ${LIB_INSTALL_DIR}/cmake/CapnProto)
```

Comment:
I guess that makes sense, This is inconsistent since I copied thiose lines from the CMake docs (http://www.cmake.org/cmake/help/v3.1/manual/cmake-packages.7.html)